src/TestxServer.test.ts

```diff
@@ -35,33 +35,41 @@ test("test start() and close() methods", async t => {
 test("should start the server after closing it", async t => {
   const server = new TestxServer({ schema: ITEM_MODEL });
   await server.start();
-  const httpUrl = await server.httpUrl();
+  const httpUrl1 = await server.httpUrl();
   const mutations = await server.getMutations();
   const queries = await server.getQueries();
 
-  await request(httpUrl, mutations.createItem, { title: "test" });
+  await request(httpUrl1, mutations.createItem, { title: "test" });
 
   await server.close();
   await t.throwsAsync(
     async () => {
-      await request(httpUrl, queries.findAllItems);
+      await request(httpUrl1, queries.findAllItems);
     },
     null,
     "Should throw an error after closing the server (ECONNREFUSED)"
   );
 
   await server.start();
-  const result = await request(httpUrl, queries.findAllItems);
+  const httpUrl2 = await server.httpUrl();
+
+  const result = await request(httpUrl2, queries.findAllItems);
```

Comment:
By using the old httpUrl instant of retrieving a new one we ensure that the url doesn't change after re-start, I think is better to revert this change

Reply:
I made this change because we can just ensure that when doing a re-start after a stop, but not after a close. The test was working before because the getAvailablePort method was not getting a random port.

Reply:
I see, you are right